src/main/java/thePackmaster/cardmodifiers/basicspack/MagicModifier.java

```diff
@@ -0,0 +1,43 @@
+package thePackmaster.cardmodifiers.basicspack;
+
+import basemod.abstracts.AbstractCardModifier;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+public class MagicModifier extends AbstractCardModifier {
+    private final int value;
+    private final boolean secondMagic;
+
+    public MagicModifier(int value){
+        this.value = value;
+        this.secondMagic = false;
+    }
+
+    public MagicModifier(int value, boolean secondMagic){
+        this.value = value;
+        this.secondMagic = secondMagic;
+    }
+
+    public boolean shouldApply(AbstractCard card) {
+        if(secondMagic && card instanceof AbstractPackmasterCard){
+            return ((AbstractPackmasterCard) card).secondMagic != 0;
+        } else return card.magicNumber != 0;
```

Comment:
Should this be != -1? 0 means it was changed, -1 means its probably not being used.